scripts/analyze-growth-retention.py

```
@@ -290,7 +353,7 @@ def main():
     rows = cur.fetchall()
     out = []
     for cw, size, w1, w2, w4, w8 in rows:
-        fmt = lambda x: f"{x} ({100*x/size:.0f}%)" if size else "0"
+        fmt = lambda x: f"{x} ({100 * x / size:.0f}%)" if size else "0"
```

Ruff (0.15.12)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `fmt` as a `def`

(E731)